src/context/extensions/helpers/processRocObject.js

```diff
@@ -29,13 +29,13 @@ export function handleResult(roc, result) {
                 'meta',
             ]),
         },
-        update: result.update,
+        context: result.context,
     };
 }
 
 /* eslint-disable no-param-reassign */
 export default function processRocObject(
-    { roc, update = {} },
+    { roc, context = {} },
```

Comment:
Could we call this `newContext` to not confuse the reader of the code since it has two contexts in the same scope?